tradewave/views.py

```diff
@@ -1090,21 +1103,112 @@ def redeem_credits(request):
         return redirect('tradewave:login', status_msg=e.message)
 
 
-# *** handler for creating a new user ***
+# *** view handler for creating a new user ***
```

Comment:
@dannypernik So I suggest we adopt this convention from now on for any template we are working on.